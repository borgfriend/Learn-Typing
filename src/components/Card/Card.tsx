import styled from "styled-components";

export const Card = styled.div`
  padding: 1.5rem;
  margin: 1rem;
  box-shadow: 0px 1px 3px 0px rgba(0, 0, 0, 0.2),
    0px 1px 1px 0px rgba(0, 0, 0, 0.14), 0px 2px 1px -1px rgba(0, 0, 0, 0.12);
  border-radius: 0.2rem;
`;
