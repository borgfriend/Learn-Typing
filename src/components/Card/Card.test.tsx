import { render } from "@testing-library/react";
import { expect, it } from "vitest";
import { Card } from "./Card";

it("renders a default button", () => {
  const { container } = render(<Card></Card>);
  expect(container).toMatchSnapshot();
});
