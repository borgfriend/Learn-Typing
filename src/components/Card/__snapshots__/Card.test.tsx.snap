// Vitest Snapshot v1

exports[`renders a default button 1`] = `
<div>
  <div
    class="sc-bcXHqe kTGhZP"
  />
</div>
`;
